test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -255,6 +255,11 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
          * - xpack.infra.logs.app_target (string)
          */
         'xpack.infra.logs (any)',
+        /**
+         * xpack.infra.featureFlags is conditional and will resolve to an object of properties
+         * - xpack.infra.featureFlags.metricsExplorerEnabled (boolean)
+         */
+        'xpack.infra.featureFlags (any)',
```

Comment:
Is there any reason not to just specifically add `xpack.infra.featureFlags.metricsExplorerEnabled`? Similarly to the apm feature flags above?
```
        'xpack.apm.featureFlags.agentConfigurationAvailable (any)',
        'xpack.apm.featureFlags.configurableIndicesAvailable (any)',
        'xpack.apm.featureFlags.infrastructureTabAvailable (any)',
        'xpack.apm.featureFlags.infraUiAvailable (any)',
        'xpack.apm.featureFlags.migrationToFleetAvailable (any)',
        'xpack.apm.featureFlags.sourcemapApiAvailable (any)',
        'xpack.apm.featureFlags.storageExplorerAvailable (any)',
```

Reply:
Update: I see now that this has to do with `featureFlags` being `schema.conditional`. As much as possible, we'd like to keep from hiding the properties exposed as settings. Could the use of `offeringBasedSchema` be moved down to the individual flag level? This would still allow you to define a different serverless default for the feature flag, but would also allow each exposed setting to be included here.

Reply:
Good point, I'll move `offeringBasedSchema`  down the hierarchy 👍